django/core/management/commands/startproject.py

```diff
@@ -19,3 +21,10 @@ def handle(self, **options):
         options["secret_key"] = SECRET_KEY_INSECURE_PREFIX + get_random_secret_key()
 
         super().handle("project", project_name, target, **options)
+
+        if target is None:
+            target = os.getcwd()
+        else:
+            target = os.path.abspath(target)
+
+        self.stdout.write(f"Success! Created {project_name} at {target}.")
```

Comment:
I don’t understand why we’re implementing this twice in the two separate commands, rather than in the `TemplateCommand` handle method called right above?

This looks more or less identical to the `startproject` output code, and it looks like we’re replicating how `target` is handled from the `TemplateCommand` logic.